src/js/stores/preferences.js

```diff
@@ -99,6 +99,10 @@ define(function (require, exports, module) {
             return this._preferences;
         },
 
+        get: function (key, unsetValue) {
```

Comment:
This isn't even a nit, it's borderline pedantry, but why not just use the existing pattern of:
```
 var preferences = this.flux.store("preferences").getState();
        if (preferences.get("showFirstLaunch", true))
...
```